convert2rhel/utils.py

```diff
@@ -473,8 +473,16 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning(
             "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
+            " \nOutput from the yum downloader call:\n%s %s" % (pkg, output)
         )
+
+        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
+            loggerinst.critical(
+                "Without thoes packages we would be unable to rollback if needed."
+                " Check to make sure that your packages are up to date."
+                " If you would rather ignore this check set the envirament variable 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK' to 1."
+            )
```

Comment:
```suggestion
                "This means we will not be able to do a complete rollback and may put the system in an broken state."
                " Check to make sure that your packages are up to date."
                " If you would rather override this check set the 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK' environment variable.\n"
            )
        else:
            loggerinst.warning(
                "'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion."
                " This may put the system in an unsupported broken state on a rollback."
            )
```

Reply:
I have reviewed this suggestion and I have added it to the most recent commit.